release_notes.md

```diff
@@ -1,118 +1,57 @@
+- Updated the mapping of the *Type* field in Threat Intel Feeds to accurately reflect the specific hash type (e.g., *FileHash-MD5*, *FileHash-SHA256*, etc.) instead of using the generic type *FileHash*.
 
-- A brand-new **Ingest Now** button on the *Summary* page allows you to install the latest outbreak-specific response solution packs instantly.
+### Enhanced CVE Details with EPSS Scores
 
-#### Improved Investigation Flow
+The following fields have been mapped into CVE records to provide enhanced threat context:  
 
-- Introducing enhanced automation with the following new schedules to streamline outbreak response tasks:
+- `EPSS Score`
+- `EPSS Percentile`
+- `Known To Be Used In Ransomware Campaigns`
 
-    - `Investigate_Outbreak-Alerts`: Automatically identify new and ongoing outbreak alerts and forward them for thorough investigation using advanced threat-hunting tools.
-    - `Outbreak_Alert-Time-Frame-Analysis`: Efficiently identify and deactivate outdated outbreak alerts based on their investigation time frame.
-    - `Outbreak_Automated-Deployment-Outbreak-Alert-Response-Solution-Pack`: Seamlessly install the latest outbreak alert solutions pack (from the past X days) with automated email notifications.
-    - `Outbreak_Ingest-Tracking-Outbreak-CVEs-and-IOCs`: Retrieve outbreak alert CVEs (KVE) from NIST and IOCs from Fortinet FortiGuard, focusing on those marked as ‘Tracking’.
+### NIST NVD Connector Action Isolation
 
-- As an investigation result, we now create incidents in place of alerts.
+The **NIST NVD Connector** action, responsible for fetching CVE details, has been isolated into a dedicated playbook named **Update CVE Details from NIST**. This playbook includes the setting `Ignore Error` set to *Yes* to handle intermittent API issues more gracefully.  
 
-#### Pluggable Threat Hunt Framework
+The **Update CVE Details from NIST** playbook is referenced as the final step in the **Find Known Exploited Vulnerabilities (KEV) CVEs** playbook. This change addresses the intermittent reliability issues with the NIST NVD APIs and ensures uninterrupted processing of CVE data.
 
-The Pluggable Threat Hunting framework integrates threat detection tools, such as FortiSIEM, FortiAnalyzer, QRadar, and Splunk, for efficient outbreak alert threat hunting and integrates the following hunting categories:
+### Key Store Enhancements
 
-- Fabric Hunting
-- Indicator of Compromise (IoC) Hunting
-- Signature Hunting
+- Key Store records associated with Outbreaks are now tagged with **`Outbreak Alert`**
 
-#### Widget Additions
+### Schedule Changes for Solution Pack Deployment
 
-We’ve integrated cutting-edge widgets to execute your playbooks with a single click and track the execution with an intuitive, user-friendly wizard.
+The crontab schedule for **Outbreak_Automated-Deployment-Outbreak-Alert-Response-Solution-Pack** has been adjusted to run **every 3 hours**. This ensures the timely deployment of new outbreak alert solution packs.  
 
```

Comment:
Outbreak Config wizard modification;

**Configure Integrations** Page Enhancements:
    - NIST National Vulnerability Database connector configuration is optional now.
    - Wizard now also verifies whether the **Mark as Default Configuration** option has been selected for all chosen integrations.